domain-cc/cc-api-controller/build.gradle

```diff
@@ -13,7 +13,6 @@ ext {
 dependencies {
   // Used by API classes
   implementation 'io.micrometer:micrometer-core'
-  implementation "gov.va.starter:exceptions"
```

Comment:
We don't use any of these exceptions: https://github.com/department-of-veterans-affairs/lighthouse-di-starter-boot/tree/release-0.1.32/exceptions/src/main/java/gov/va/starter/boot/exception